dubinc/dub
# Fix : Images getting cropped when help center is active in small devices

Fixes #1428 

# Issue
In small devices when opening the help center the top level div of the [New Background](https://github.com/dubinc/dub/blob/main/apps/web/ui/shared/new-background.tsx) Component was shrinking its size down and was not starting itself from top:0, therefore the images inside that was also starting from that level, which to user, seems like the image is getting cropped.
## Images
<img width="1049" alt="Screenshot 2024-10-20 at 11 08 06 AM" src="https://github.com/user-attachments/assets/03c58dc7-0b8f-4a7a-9102-b43d1220aedb">
<img width="1043" alt="Screenshot 2024-10-20 at 11 12 19 AM" src="https://github.com/user-attachments/assets/71103ff5-4030-4264-a729-fec62bfd46d0">

# What does this PR do

changing its position from absolute to fixed sticks it to the top always, therefore the image will start from the top and look normal

## Images
<img width="1053" alt="Screenshot 2024-10-20 at 11 10 54 AM" src="https://github.com/user-attachments/assets/5b437b9e-aef6-4443-9683-4d4f9d2e41ba">
<img width="1057" alt="Screenshot 2024-10-20 at 11 11 39 AM" src="https://github.com/user-attachments/assets/b8d6b665-860b-4a00-b5e8-c2e434963f59">
<img width="1041" alt="Screenshot 2024-10-20 at 11 14 12 AM" src="https://github.com/user-attachments/assets/de99c9f0-fd53-4d44-9d21-7224aa23a276">

# How to test this

1. Visit https://app.dub.co/onboarding/welcome
2. Check in Mobile or Inspect and Choose Mobile Screens
3. Click the Help Center Icon ( ? ) at the bottom left

# Oss

As you can see many people have already tried fixing it and it was not a easy to find issue so I would request you to award a bit more point for this fix, thanks in advance!
